gcc.spec

```
@@ -139,6 +142,44 @@ CFLAGS="${CONF_FLAGS}" ./configure --static --prefix=%{i}/tmp/sw
 make %{makeprocesses}
 make install
 
+# Build zstd (required for compressing LTO information)
+cd ../zstd-%{zstdVersion}
+rm -f ./config.{sub,guess}
+%get_config_sub ./config.sub
+%get_config_guess ./config.guess
+chmod +x ./config.{sub,guess}
+
+cmake build/cmake \
+ -DZSTD_BUILD_CONTRIB:BOOL=OFF \
+ -DZSTD_BUILD_STATIC:BOOL=OFF \
+ -DZSTD_BUILD_SHARED:BOOL=ON \
+ -DZSTD_BUILD_TESTS:BOOL=OFF \
+ -DCMAKE_BUILD_TYPE=Release \
+ -DZSTD_BUILD_PROGRAMS:BOOL=OFF \
+ -DZSTD_LEGACY_SUPPORT:BOOL=OFF \
+ -DCMAKE_INSTALL_PREFIX:STRING=%{i}/tmp/sw \
+ -DCMAKE_INSTALL_LIBDIR:STRING=lib \
+ -Dzstd_VERSION:STRING=%{zstdVersion}
+
+make %{makeprocesses} VERBOSE=1
+make install
+
+# Build zstd (for runtime)
+cmake build/cmake \
+ -DZSTD_BUILD_CONTRIB:BOOL=OFF \
+ -DZSTD_BUILD_STATIC:BOOL=OFF \
+ -DZSTD_BUILD_SHARED:BOOL=ON \
+ -DZSTD_BUILD_TESTS:BOOL=OFF \
+ -DCMAKE_BUILD_TYPE=Release \
+ -DZSTD_BUILD_PROGRAMS:BOOL=OFF \
+ -DZSTD_LEGACY_SUPPORT:BOOL=OFF \
+ -DCMAKE_INSTALL_PREFIX:STRING=%{i} \
+ -DCMAKE_INSTALL_LIBDIR:STRING=lib \
+ -Dzstd_VERSION:STRING=%{zstdVersion}
+
+make %{makeprocesses} VERBOSE=1
+make install
+
```

```suggestion
#Build and install zstd static library
make -C ../zstd-%{zstdVersion}/lib %{makeprocesses} \
  install-static install-includes prefix=%{i}/tmp/sw \
  CPPFLAGS="-fPIC" CFLAGS="-fPIC"
```

try this locally and see if you can build gcc with zstd